README.md

```diff
@@ -1,5 +1,6 @@
 # Rust ABCI
-A rust implementation of the ABCI protocol for Tendermint.
+A rust implementation of the ABCI protocol for Tendermint. This has been heavily modified to be consistent and compliant with
```

Comment:
This looks like a changelog entry. Don't think we should put this in the top of the readme.

Reply:
OK, I'll revert it